pkg/core/tekton/constants.go

```diff
@@ -0,0 +1,21 @@
+package tekton
+
+const (
+	pipelineRunPrefix         = "fuseml-"
+	pipelineRunServiceAccount = "staging-triggers-admin"
+	workspaceAccessMode       = "ReadWriteOnce"
+	workspaceSize             = "2Gi"
+	inputTypeCodeset          = "codeset"
+	codesetWorkspaceName      = "source"
+	builderTaskName           = "kaniko"
+	cloneTaskName             = "clone"
+	codesetNameParam          = "codeset-name"
+	codesetVersionParam       = "codeset-version"
+	codesetURLParam           = "codeset-url"
+	fuseMLRegistry            = "registry.fuseml-registry"
+	fuseMLRegistryLocal       = "127.0.0.1:30500"
+	imageParamName            = "IMAGE"
+	stepOutputVarName         = "TASK_RESULT"
+	inputsVarPrefix           = "FUSEML_"
+	stepDefaultCmd            = "run"
```

Comment:
Could you please remind me why this is needed ? All I remember is that it's a workaround for a Tekton limitation regarding entrypoints and I need to at least create an issue for it.

Reply:
I think you already did: https://github.com/fuseml/fuseml/issues/65

This is a workaround for that issue as for Tekton to be able to call the container entrypoint it needs access to the registry (only possible when image is set to registry.fuseml-registry)and at the same time the kubernetes nodes also needs to resolve the registry to be able to pull the image (only possible through 127.0.0.1:30500). Basically one requirement excludes the other.

The workaround consists in having a `run` command in the container image (the entrypoint) and setting the tekton step to explicitly run that command.

Reply:
Awesome, thanks for the clarification. I suspected the multiple hostname issue was somehow related to this, but didn't know how. I'll update it to reflect this info.